packages/playwright-test/src/runner.ts

```diff
@@ -424,6 +444,15 @@ export class Runner {
       const testGroups = createTestGroups(projectSuites, config.workers);
       concurrentTestGroups.push(testGroups);
     }
+    return { rootSuite, concurrentTestGroups };
+  }
+
+  private async _run(options: RunOptions): Promise<FullResult> {
```

Comment:
Why this method split? Now `7. Fail when no tests` seems strange :)

Reply:
For better readability. Removed numbers.